protos/protos/public/modeldb/versioning/VersioningService.proto

```diff
@@ -418,6 +418,45 @@ message RevertRepositoryCommitsRequest {
     }
 }
 
+message FindRepositories {
+    repeated uint64 repo_ids = 1;
```

Comment:
Add comment on what happens if empty or not. Does it search all? Does it search a subset?

Reply:
if `repo_ids ` not present in request then the backend returns all the Repositories. I will add comments on the protos.

Reply:
If any of the request parameters are not set then backend will return all the repositories. the query like: ` FROM RepositoryEntity repo ORDER BY repo.date_updated DESC`